### PR TITLE
Added support for bedrock servers

### PR DESCRIPTION
**Pull request by AlexBrin**

### CODE REVIEW

File: pkg/bedrock/client.go
Comment (xrjr) [inline pkg/bedrock/client.go]: Declare errors globally

File: pkg/bedrock/client.go
Comment (xrjr) [inline pkg/bedrock/client.go:80]: I can't merge code that can break

File: pkg/bedrock/bedrock.go
Comment (xrjr) [inline pkg/bedrock/bedrock.go]: We don't want this library to print to standard output

File: pkg/bedrock/client.go
Comment (xrjr) [inline pkg/bedrock/client.go:89]: This comment isn't really reassuring. Is this method robust ?

File: pkg/bedrock/client.go
Comment (xrjr) [inline pkg/bedrock/client.go]: Declare errors globally

File: pkg/bedrock/client.go
Comment (xrjr) [inline pkg/bedrock/client.go]: Declare errors globally

**Review by xrjr — CHANGES_REQUESTED** [review]
This PR adds a great feature, but needs some adjustments before being merged.

I really appreciate your contribution and the way you built your implementation using existing methods and naming conventions. Thanks a lot !

File: pkg/bedrock/client.go
Comment (AlexBrin) [reply pkg/bedrock/client.go]: Sorry, I didn't pay attention to the existing errors. Declared everything globally

File: pkg/bedrock/bedrock.go
Comment (AlexBrin) [reply pkg/bedrock/bedrock.go]: Done

File: pkg/bedrock/client.go
Comment (AlexBrin) [reply pkg/bedrock/client.go:80]: This will not lead to a fatal error. In this case, the error lies only in incorrect parsing
Maybe there are ideas how this can be handled?

File: pkg/bedrock/client.go
Comment (AlexBrin) [reply pkg/bedrock/client.go:89]: Unfortunately, this is the only method of analyzing the server response. It comes in the format "MCPE;motd of the server;protocol version;etc...". But due to the simplicity of the server software, everyone who is not lazy makes their forks and can change the answer as they like. There is no single standard

**Comment by xrjr** [comment]
I can't merge it to master yet. I'll merge it to a new branch `add-support-for-bedrock-servers` and try to improve parsing. Thank you for your contribution.

**Pull request merged**
